site/app/libraries/DiffViewer.php

```diff
@@ -398,13 +404,12 @@ private function getDisplay($lines, $type="expected") {
                 $html .= "\t</div>\n";
             }
         }
-
         /*
          * Run through every line, starting a highlight around any group of mismatched lines that exist (whether
          * there's a difference on that line or that the line doesn't exist.
          */
         for ($i = 0; $i < count($lines); $i++) {
-            $j = $i + 1;
+        	$j = $i + 1;
```

Comment:
You really should configure your editor to not use tabs. There's a couple other instances where you've used tabs in this file.